docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,48 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+EPID attestation support has been discontinued in newer processors (Intel Xeon
+3rd generation onwards). All nodes using EPID attestation should migrate to DCAP
+Attestation.
+
+:::info
+
+Validators, please hold for further instructions. We will coordinate the
+migration in phases to ensure that during the migration the network remains
+operational.
```

Comment:
You mean compute node operators?

Reply:
Yeah, well they need to be also validators. But yes, compute node operators is more appropriate.